taskcluster/scripts/toolchain/android-gradle-dependencies/before.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -vex
+
+set -x -e
+
+echo "running as" $(id)
+
+: WORKSPACE ${WORKSPACE:=/builds/worker/workspace}
+
+set -v
+
+mkdir -p ${NEXUS_WORK}/conf
+cp /builds/worker/checkouts/src/taskcluster/scripts/toolchain/android-gradle-dependencies/nexus.xml ${NEXUS_WORK}/conf/nexus.xml
+
+RUN_AS_USER=worker /opt/sonatype/nexus/bin/nexus restart
+
+# Wait "a while" for Nexus to actually start.  Don't fail if this fails.
+wget --quiet --retry-connrefused --waitretry=2 --tries=100 \
+  http://localhost:8081/nexus/service/local/status || true
+rm -rf status
+
+# It's helpful when debugging to see the "latest state".
+curl http://localhost:8081/nexus/service/local/status || true
+
+# Verify Nexus has actually started.  Fail if this fails.
+curl --fail --silent --location http://localhost:8081/nexus/service/local/status | grep '<state>STARTED</state>'
```

Comment:
I wonder if this should be something like
```shell
if ! curl ... | grep --quiet '...'; then
   echo "Nexus failed to start"
fi
```
In any case, since this is copied from upstream, we can leave it as is. It probably makes the most sense to do this upstream, and then update here.

Reply:
I'll ask Nick Alexander when he's back. I don't have the context about this current choice. Agreed to leave it as is, for now.